### ISSUE TITLE
pkg_rpm error: /usr/lib/rpm/check-files Installed (but unpackaged) file(s) found

### ISSUE THREAD

**Issue by mmikitka** [bazelbuild/rules_pkg#107]
Since upgrading from pkg_rpm in @bazel_tools to rules_pkg, I encountered the following error, without changes to my SPEC file:

```
Checking for unpackaged file(s): /usr/lib/rpm/check-files /tmp/tmp4k4_jcvn/BUILD
error: Installed (but unpackaged) file(s) found:
   /bazel-out/k8-dbg/bin/myfile

RPM build errors:
    Installed (but unpackaged) file(s) found:
   /bazel-out/k8-dbg/bin/myfile
```

The cause appears to be a change to the following code (--buildroot points to the `BUILD` folder instead of the `BUILDROOT` folder)

```
   buildroot = os.path.join(dirname, RpmBuilder.BUILD_DIR)
    args = [
        self.rpmbuild_path,
        '--define',
        '_topdir %s' % dirname,
        '--define',
        '_tmppath %s/TMP' % dirname,
        '--bb',
        '--buildroot=%s' % buildroot,
        self.spec_file,
    ]
    os.environ['RPM_BUILD_ROOT'] = buildroot
    env = {
        'LANG': 'C',
    }
    if override_lib_file:
        env['LD_PRELOAD'] = os.path.join(dirname,
                                         RpmBuilder.LIB_DIR,
                                         os.path.basename(override_lib_file))

```

**Comment by mmikitka** [comment]
PR is forthcoming

**Comment by aiuto** [comment]
Believed fixed by #108
